index.js

```diff
@@ -4,6 +4,9 @@ const $ = require('sanctuary-def');
 const Z = require('sanctuary-type-classes');
 
 
+//    stripNamespace :: String -> String
+const stripNamespace = s => s.slice(s.indexOf('/') + 1);
```

Comment:
What should happen if input is `my-package/sub-package/Point` ? maybe we shuold use `lastIndexOf`?